lib/graphql/resolvers.ts

```diff
@@ -7,25 +7,27 @@ import { permits, createPermit } from '@lib/permits/resolvers';
 import { IFieldResolver } from 'graphql-tools'; // GraphQL field resolver
 import { Context } from '@lib/context'; // Context type
 import { dateScalar } from '@lib/scalars'; // Custom date scalar implementation
+import { authenticate } from '@lib/auth-guards';
+import { Role } from '@lib/graphql/types';
 
 // Resolver type
 export type Resolver<P = undefined> = IFieldResolver<P, Context>;
 
 const resolvers = {
   Query: {
-    meta,
-    applicants,
-    employees,
-    physicians,
-    applications,
-    permits,
+    meta: meta,
```

Comment:
```suggestion
    meta,
```
Small nit but I think it reduces redundancy.